generated/rflx-rflx_generic_types.ads

```diff
@@ -42,17 +44,16 @@ package RFLX.RFLX_Generic_Types is
    type Offset is mod 8;
 
    generic
-      type Value is (<>);
+      type Value is mod <>;
    function Extract (Data : Bytes;
                      Ofst : Offset) return Value with
      Pre =>
        ((Offset'Pos (Ofst) + Value'Size - 1) / Byte'Size < Data'Length
         and then (Offset'Pos (Ofst) + Value'Size - 1) / Byte'Size <= Natural'Size
-        and then Natural (((Offset'Pos (Ofst) + Value'Size - 1) / Byte'Size) * Byte'Size) < Long_Integer'Size - 1
         and then (Byte'Size - Natural (Offset'Pos (Ofst) mod Byte'Size)) < Long_Integer'Size - 1);
 
    generic
-      type Value is (<>);
+      type Value is mod <>;
    procedure Insert (Val  :        Value;
```

Comment:
Same as above (Val -> Value)

Reply:
How should this work? `Value` is not part of `RFLX_Generic_Types`, so `RFLX_Generic_Types.Value` doesn't work.

Reply:
My bad. I didn't realize Value is a generic type.